### ISSUE TITLE
Notification on event screen

### ISSUE THREAD

**Issue by Druide13** [tadzik/harbour-daily#3]
Ah, here we go! :)
I thought somthing like this: A notification appears on event screen, with maybe showing the first (undone) task - or simply the text "Undone tasks to do", and if you tap on it, the daily app would start and show your tasks. If all tasks are done, the notification vanishes for the rest of the day.

**Comment by tadzik** [comment]
Yep, that sounds like a useful reminder indeed; I'll do some experimenting with it and see how it turns out :)

**Comment by Druide13** [comment]
Great! Thank you very much!
